{"uid":"429ce507d4f52582","name":"test_invalid_config_and_recovery_after_fixing_it","fullName":"tests.integration.test_backups#test_invalid_config_and_recovery_after_fixing_it","historyId":"eea1c49c5c49dca334567c1ed4ab365e","time":{"start":1724897626031,"stop":1724897626031,"duration":0},"description":"Test that the charm can handle invalid and valid backup configurations.","descriptionHtml":"<p>Test that the charm can handle invalid and valid backup configurations.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 844, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 873, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 159, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 517, in setup\n    raise exc\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1635, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 686, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 521, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 606, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1076, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 228, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1724895714768,"stop":1724895714768,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"github_secrets","time":{"start":1724895714768,"stop":1724895714769,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1724895714768,"stop":1724895714768,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1724895714769,"stop":1724895714962,"duration":193},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"cloud_configs","time":{"start":1724895714963,"stop":1724895715038,"duration":75},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1724897626032,"stop":1724897626032,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 228, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Test that the charm can handle invalid and valid backup configurations.","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 85, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 844, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 873, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 82, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 159, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 517, in setup\n    raise exc\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1635, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 686, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 521, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 606, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1076, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 228, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 205, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","steps":[],"attachments":[{"uid":"6317ab0b3657e24b","name":"log","source":"6317ab0b3657e24b.txt","type":"text/plain","size":3553}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"cloud_configs::finalizer","time":{"start":1724897626034,"stop":1724897627296,"duration":1262},"status":"broken","statusMessage":"botocore.exceptions.NoCredentialsError: Unable to locate credentials\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/test_backups.py\", line 96, in cloud_configs\n    for bucket_object in bucket.objects.filter(Prefix=config[\"path\"].lstrip(\"/\")):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 79, in __iter__\n    for page in self.pages():\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/boto3/resources/collection.py\", line 169, in pages\n    for page in pages:\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 269, in __iter__\n    response = self._make_request(current_kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/paginate.py\", line 357, in _make_request\n    return self._method(**current_kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 565, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1001, in _make_api_call\n    http, parsed_response = self._make_request(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/client.py\", line 1027, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 119, in make_request\n    return self._send_request(request_dict, operation_model)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 198, in _send_request\n    request = self.create_request(request_dict, operation_model)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/endpoint.py\", line 134, in create_request\n    self._event_emitter.emit(\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 412, in emit\n    return self._emitter.emit(aliased_event_name, **kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 256, in emit\n    return self._emit(event_name, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/hooks.py\", line 239, in _emit\n    response = handler(**kwargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 105, in handler\n    return self.sign(operation_name, request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/signers.py\", line 199, in sign\n    auth.add_auth(request)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/botocore/auth.py\", line 418, in add_auth\n    raise NoCredentialsError()\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1724897627297,"stop":1724897627694,"duration":397},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1724897627695,"stop":1724897627695,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1724897627695,"stop":1724897627695,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1724897627695,"stop":1724897627695,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(2)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_backups"},{"name":"host","value":"github-runner"},{"name":"thread","value":"10082-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_backups"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"393403dbb6c21511","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":78,"passed":0,"unknown":3,"total":81},"items":[{"uid":"a63a47ef5687e27","reportUrl":"../156//#testresult/a63a47ef5687e27","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724810820220,"stop":1724810820220,"duration":0}},{"uid":"fe9a056de2bc8953","reportUrl":"../155//#testresult/fe9a056de2bc8953","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724722723109,"stop":1724722723109,"duration":0}},{"uid":"1d562e70dc2ef5c8","reportUrl":"../154//#testresult/1d562e70dc2ef5c8","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724638104476,"stop":1724638104476,"duration":0}},{"uid":"b1ad0debdd213bf0","reportUrl":"../153//#testresult/b1ad0debdd213bf0","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724551850104,"stop":1724551850104,"duration":0}},{"uid":"27cc1fbad8089c00","reportUrl":"../152//#testresult/27cc1fbad8089c00","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724465226313,"stop":1724465226313,"duration":0}},{"uid":"5e91ec5d64f3475d","reportUrl":"../151//#testresult/5e91ec5d64f3475d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724378762111,"stop":1724378762111,"duration":0}},{"uid":"60f79b54d0047d69","reportUrl":"../150//#testresult/60f79b54d0047d69","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724292728361,"stop":1724292728361,"duration":0}},{"uid":"db6deceaebeca255","reportUrl":"../149//#testresult/db6deceaebeca255","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724206024311,"stop":1724206024311,"duration":0}},{"uid":"3b27d4b7b4b56018","reportUrl":"../148//#testresult/3b27d4b7b4b56018","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724119715098,"stop":1724119715098,"duration":0}},{"uid":"84fe177f746bf121","reportUrl":"../147//#testresult/84fe177f746bf121","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1724039608410,"stop":1724039608410,"duration":0}},{"uid":"96a3cea1a183476e","reportUrl":"../146//#testresult/96a3cea1a183476e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723946794319,"stop":1723946794319,"duration":0}},{"uid":"e0d80cb1251a32b5","reportUrl":"../145//#testresult/e0d80cb1251a32b5","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723860698310,"stop":1723860698310,"duration":0}},{"uid":"316496706fee971d","reportUrl":"../143//#testresult/316496706fee971d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723688228523,"stop":1723688228523,"duration":0}},{"uid":"2b1ff3720353d201","reportUrl":"../142//#testresult/2b1ff3720353d201","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723601939914,"stop":1723601939914,"duration":0}},{"uid":"ceecd60b14e2122","reportUrl":"../141//#testresult/ceecd60b14e2122","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723515465383,"stop":1723515465383,"duration":0}},{"uid":"2cf362610f2dd752","reportUrl":"../140//#testresult/2cf362610f2dd752","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723429013720,"stop":1723429013720,"duration":0}},{"uid":"2d8c0ea8acc2f14e","reportUrl":"../139//#testresult/2d8c0ea8acc2f14e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723342297073,"stop":1723342297073,"duration":0}},{"uid":"6862b0dbdfbe72aa","reportUrl":"../138//#testresult/6862b0dbdfbe72aa","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723256071673,"stop":1723256071673,"duration":0}},{"uid":"335aaa76ab44c40","reportUrl":"../137//#testresult/335aaa76ab44c40","status":"unknown","time":{}},{"uid":"395112dfe9e76be6","reportUrl":"../136//#testresult/395112dfe9e76be6","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1723083939207,"stop":1723083939207,"duration":0}}]},"tags":["asyncio","group(2)"]},"source":"429ce507d4f52582.json","parameterValues":[]}